doc/Language/classtut.pod6

```diff
@@ -340,91 +284,40 @@ C<.an-attribute()> syntax. See also
 L<the C<is rw> trait on classes|/language/typesystem#trait_is_rw> for examples
 on how this works on the whole class.
 
+=head2 Class variables
+
 X<|class variables>
 A class declaration can also include I<class variables>, which are variables
 whose value is shared by all instances, and can be used for things like
-counting the number of instantiations or any other shared state.
-Class variables use the same syntax as the rest of the attributes, but are
-declared as C<my> or C<our>, depending on the scope; C<our> variables will be
-shared by subclasses, since they have package scope.
+counting the number of instantiations or any other shared state. So I<class variables> act similar to
+I<static> variables known from other programming languages.
+They look the same as normal (non class) lexical variables (and in fact they are the same):
 
 =begin code
 class Str-with-ID is Str {
     my $counter = 0;
-    our $hierarchy-counter = 0;
     has Str $.string;
-    has Int $.ID;
+    has Int $.ID is built(False);
 
-    method TWEAK() {
+    submethod TWEAK() {
```

Comment:
Yep, that's a good one. We should definitely change that. I'll probably create an issue.